packages/containers/src/List/List.connect.js

```diff
@@ -4,42 +4,43 @@ import { cmfConnect } from '@talend/react-cmf';
 import Container, { DEFAULT_STATE } from './List.container';
 import {
 	configureGetFilteredItems,
-	configureGetPagination,
 	configureGetPagedItems,
+	configureGetPagination,
+	getCollectionItems,
 } from './selector';
 
 function componentId(ownProps) {
 	return ownProps.collectionId;
 }
 
-function getItems(state, config) {
-	const items = configureGetFilteredItems(config)(state);
-
-	return items || new List();
-}
-
-function getPagedItems(state, config) {
-	const items = configureGetPagedItems(config)(state);
+function getPagedItems(state, config, filteredItems) {
+	const items = configureGetPagedItems(config, filteredItems)(state);
 
 	return items || new List();
 }
 
 export function mapStateToProps(state, ownProps, cmfProps) {
 	const props = {};
-	const config = {
+	let filteredItems;
+	const collectionItems = getCollectionItems(state, ownProps.collectionId);
+
+	props.config = {
 		collectionId: ownProps.collectionId,
-		items: ownProps.items,
+		items: collectionItems || ownProps.items || new List(),
```

Comment:
If there is no items, to attach a new `List`. It means that it's a new ref at each render.
Can you remove that and replace it by a default props in container please ?